tests/unit/components/filter-selector/FilterSelector.spec.jsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react'
+import user from '@testing-library/user-event'
+import FilterSelector from '~/components/filter-selector/FilterSelector'
+
+vi.mock('simplebar-react', () => {
+  return {
+    __esModule: true,
+    default: function MockedSimpleBar(props) {
+      return <div>{props.children}</div>
+    }
+  }
+})
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key) => {
+      const translations = {
+        'common.noItems': 'No items found'
+      }
+      return translations[key] || key // Return the custom translation or the key itself
+    }
+  })
+}))
+describe('FilterSelector', () => {
+  const selectedItems = []
+  test('FilterSelector component renders correctly', () => {
+    render(<FilterSelector selectedItems={selectedItems} />)
+    const el = screen.getByRole('button')
+    expect(el).toBeInTheDocument()
+  })
+
+  test('renders no items after clicking', async () => {
+    render(<FilterSelector selectedItems={[]} />)
```

Comment:
if you render this component with the same props for each test, better move this render into forEach block and remove from tests